coding_style/README.adoc

```diff
@@ -161,6 +165,9 @@ Rationale:: https://yaml.org/type/bool.html[YAML 1.1] allows all variants wherea
   your task will likely need options such as `changed_when:` and maybe `check_mode:`).
 * Anytime `command` or `shell` modules are used, add a comment in the code with justification to help with future maintenance.
 * Use the `| bool` filter when using bare variables (expressions consisting of just one variable reference without any operator) in `when`.
+* No single task file should contain more than 30 tasks.  Complex task files should be broken down into discrete parts.
```

Comment:
Any explanation of how we  came to the number 30 tasks? Based on how the role is implemented 30 tasks can be written in less than 100 lines and on the other side 5 tasks can take more than 500 lines (eg. network tasks)

Reply:
I think its arbitrary, but the idea being is that a role could have 1000 tasks, Just that the task files should be broken down into manageable files for different tasks.

Reply:
I believe it was an arbitrary value, this is taken from in the internal draft from last month.  I don't have especially strong feelings about exactly what number goes here.

Reply:
Few comments on this one:
- I don't think we want a concrete number, just a principle, each team can decide what works for them.
- This statement does belong here, it's more a role than code topic, but actually I think we have it already covered in https://github.com/redhat-cop/automation-good-practices/blob/main/structures/README.adoc (the "component" rationale, splitting big tasks into manageable bits).